Makefile

```diff
@@ -44,84 +42,44 @@ ifndef BAZEL_VERSION
-
-.PHONY: dep-install
-dep-install: ## Force install go dependencies
-	${DEP} ensure
-	bazel run //:gazelle $(BAZEL_ARGS)
-
-.PHONY: gazelle
-gazelle: ## Run Bazel Gazelle
-	bazel run //:gazelle $(BAZEL_ARGS)
-
-.PHONY: check-install
-check-install: ## Checks that you've installed this repository correctly
-	@./scripts/check-install.sh
-
-.PHONY: manager
-manager: generate  ## Build manager binary.
-	bazel build //cmd/manager $(BAZEL_ARGS)
-	install bazel-bin/cmd/manager/${BINARYPATHPATTERN}/manager $(shell go env GOPATH)/bin/aws-manager
-
-.PHONY: clusterctl
-clusterctl: generate ## Build clusterctl binary.
-	bazel build --workspace_status_command=./hack/print-workspace-status.sh //cmd/clusterctl $(BAZEL_ARGS)
-	install bazel-bin/cmd/clusterctl/${BINARYPATHPATTERN}/clusterctl $(shell go env GOPATH)/bin/clusterctl
-
-.PHONY: clusterawsadm
-clusterawsadm: dep-ensure ## Build clusterawsadm binary.
-	bazel build --workspace_status_command=./hack/print-workspace-status.sh //cmd/clusterawsadm $(BAZEL_ARGS)
-	install bazel-bin/cmd/clusterawsadm/${BINARYPATHPATTERN}/clusterawsadm $(shell go env GOPATH)/bin/clusterawsadm
 
-.PHONY: release-artifacts
-release-artifacts: ## Build release artifacts
-	bazel build --platforms=@io_bazel_rules_go//go/toolchain:linux_amd64 //cmd/clusterctl //cmd/clusterawsadm
-	bazel build --platforms=@io_bazel_rules_go//go/toolchain:darwin_amd64 //cmd/clusterctl //cmd/clusterawsadm
-	bazel build //cmd/clusterctl/examples/aws
-	mkdir -p out
-	install bazel-bin/cmd/clusterawsadm/darwin_amd64_pure_stripped/clusterawsadm out/clusterawsadm-darwin-amd64
-	install bazel-bin/cmd/clusterawsadm/linux_amd64_pure_stripped/clusterawsadm out/clusterawsadm-linux-amd64
-	install bazel-bin/cmd/clusterctl/darwin_amd64_pure_stripped/clusterctl out/clusterctl-darwin-amd64
-	install bazel-bin/cmd/clusterctl/linux_amd64_pure_stripped/clusterctl out/clusterctl-linux-amd64
-	install bazel-bin/cmd/clusterctl/examples/aws/aws.tar out/cluster-api-provider-aws-examples.tar
+## --------------------------------------
+## Testing
+## --------------------------------------
 
-.PHONY: test verify
+.PHONY: test
 test: generate verify ## Run tests
 	bazel test --nosandbox_debug //pkg/... //cmd/... $(BAZEL_ARGS)
 
-verify:
-	./hack/verify_boilerplate.py
-
-.PHONY: copy-genmocks
-copy-genmocks: ## Copies generated mocks into the repository
-	cp -Rf bazel-genfiles/pkg/* pkg/
+## --------------------------------------
+## Docker
+## --------------------------------------
 
 .PHONY: docker-build
 docker-build: generate ## Build the production docker image
 	bazel run //cmd/manager:manager-image $(BAZEL_DOCKER_ARGS)
 
 .PHONY: docker-build-dev
 docker-build-dev: generate ## Build the development docker image
-	bazel run //cmd/manager:manager-image-dev  $(BAZEL_DOCKER_ARGS)
+	bazel run //cmd/manager:manager-image-dev $(BAZEL_DOCKER_ARGS)
 
 .PHONY: docker-push
-docker-push: generate ## Push production docker image
+docker-push: ## Push production docker image
```

Comment:
```suggestion
docker-push: generate ## Push production docker image
```
?

Reply:
I don't think this is/was needed, it's already part of the docker-build process. I outlined the reasoning in the PR description.